packages/library-sync/src/iosMain/kotlin/io/realm/mongodb/internal/HttpClientCache.kt

```diff
@@ -0,0 +1,14 @@
+package io.realm.mongodb.internal
+
+import io.ktor.client.HttpClient
+
+/**
+ * Cache HttpClient on iOS.
+ * https://github.com/realm/realm-kotlin/issues/480 only seem to be a problem on macOS.
+ */
+actual class HttpClientCache actual constructor(timeoutMs: Long) {
+    private val client = createClient(timeoutMs)
```

Comment:
isn't iOS also experiencing the same behaviour as in macOS that needs to create a new client each time as a workaround?